demo/date-picker-basic-demos.html

```diff
@@ -6,141 +6,47 @@
       }
     </style>
 
-
-    <h3>Date picker</h3>
-    <vaadin-demo-snippet id="date-picker-basic-demos-label-and-placeholder-attributes-together">
-      <template preserve-content>
-        <vaadin-date-picker label="Date" placeholder="Pick a date">
-        </vaadin-date-picker>
-      </template>
-    </vaadin-demo-snippet>
-
-
-    <h3>Pre-selected value</h3>
-    <vaadin-demo-snippet id="date-picker-basic-demos-pre-selected-value">
+    <h3>Basic usage</h3>
+    <vaadin-demo-snippet>
       <template preserve-content>
-        <vaadin-date-picker label="Birthday" value="1980-08-14">
-        </vaadin-date-picker>
+        <vaadin-date-picker label="Label"></vaadin-date-picker>
+        <vaadin-date-picker  placeholder="Placeholder"></vaadin-date-picker>
```

Comment:
Nit: Extra space between tag name and attribute.

Reply:
Done.